connect/runtime/src/main/java/org/apache/kafka/connect/runtime/TopicCreationConfig.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.connect.runtime;
+
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.common.config.ConfigException;
+import org.apache.kafka.connect.util.TopicAdmin;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.regex.PatternSyntaxException;
+
+public class TopicCreationConfig {
+
+    public static final String DEFAULT_TOPIC_CREATION_PREFIX = "topic.creation.default.";
+    public static final String DEFAULT_TOPIC_CREATION_GROUP = "default";
+
+    public static final String INCLUDE_REGEX_CONFIG = "include";
+    private static final String INCLUDE_REGEX_DOC = "A list of strings that represent regular "
+            + "expressions that may match topic names. This list is used to include topics that "
+            + "match their values and apply this group's specific configuration to the topics "
+            + "that match this inclusion list.";
```

Comment:
This still is unclear, particularly what "match their values" means. Maybe:
```suggestion
    private static final String INCLUDE_REGEX_DOC = "A list of regular expression literals "
            + "used to match the names topics used by the source connector. This list is used "
            + "to include topics that should be created using the topic settings defined by this group.";
```

Reply:
I assume you mean to `match the topic names` 
I added what I had on the kip here. I agree it's a bit convoluted.

Reply:
Accepting and will fix typo in a follow up commit.